From ec7220afdcf2c0bbf9a8cd31ee4fdcd3684564de Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Tue, 15 Oct 2013 16:44:18 +0800
Subject: [PATCH 256/602] ENGR00283505 fix HDMI display lost issue when chip
 too hot

HDMI display lost when do long time 1080p recording,
root cause is IPU report error when chip too hot and
gpu downgrade clock to 1/64, fix by set the minimum
gpu 3D clock to 3/64.

IPU error log:
W/imx6.gralloc( 2360): FBIOPAN_DISPLAY failed: Connection timed out
mxc_sdc_fb mxc_sdc_fb.0: timeout when waiting for flip irq

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 imx6/etc/init.rc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 750b868..2ccded6 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -414,6 +414,9 @@ on boot
 #  Enable adb security for JB4.2.2
     setprop ro.adb.secure 1
 
+#  Set GPU 3D minimum clock to 3/64
+   write /sys/module/galcore/parameters/gpu3DMinClock 3
+
 # set this to not let jni check crash, just warnning message.
     setprop dalvik.vm.jniopts warnonly
 
-- 
1.8.0

